packages/experience/src/pages/MfaBinding/BackupCodeBinding/index.tsx

```diff
@@ -64,11 +66,14 @@ const BackupCodeBinding = () => {
         </div>
         <Button
           title="action.continue"
-          onClick={() => {
-            void sendMfaPayload({
+          isLoading={isSubmitting}
+          onClick={async () => {
+            setIsSubmitting(true);
+            await sendMfaPayload({
               flow: UserMfaFlow.MfaBinding,
               payload: { type: MfaFactor.BackupCode },
             });
+            setIsSubmitting(false);
```

Comment:
Same here. Please check if a try-catch clause is needed

Reply:
All error will be handled inside the sendMfaPayload function.